src/main/java/com/sigopt/model/Observation.java

```diff
@@ -115,5 +115,10 @@ public Builder task(Task task) {
             this.obj.set("task", task);
             return this;
         }
+
+        public Builder trainingRun(String training_run) {
```

Comment:
missing getter?

Reply:
Ah, actually shouldn't be here missed removing it.